src/remote/server/tauri-driver.api.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from 'express';
+import { TauriDriverUtility } from '../../utils/tauri-driver.util';
+import { HttpStatus } from './http-status';
+
+
+export class TauriDriverAPI {
+    public static async start(req: Request, res: Response): Promise<any> {
+        try {
+            TauriDriverUtility.start();
+            return res.status(HttpStatus.NO_CONTENT).send();
+        } catch (err) {
+            return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({"error": err}).send();
```

Comment:
Could it be used the generic ErrorHandlerAPI class in here also for the error handling? The same comment applies to the method below.

Reply:
Applied